core/services/ocrcommon/discoverer_database.go

```diff
@@ -14,35 +15,56 @@ import (
 
 var _ ocrnetworking.DiscovererDatabase = &DiscovererDatabase{}
 
+const (
+	// ocrDiscovererTable is the name of the table used to store OCR announcements
+	ocrDiscovererTable = "ocr_discoverer_announcements"
+	// don2donDiscovererTable is the name of the table used to store DON2DON announcements
+	don2donDiscovererTable = "don2don_discoverer_announcements"
+)
+
 type DiscovererDatabase struct {
-	ds     sqlutil.DataSource
-	peerID string
+	ds        sqlutil.DataSource
+	peerID    string
+	tableName string
+}
+
+func NewOCRDiscovererDatabase(ds sqlutil.DataSource, peerID string) *DiscovererDatabase {
+	return &DiscovererDatabase{
+		ds:        ds,
+		peerID:    peerID,
+		tableName: ocrDiscovererTable,
+	}
 }
 
-func NewDiscovererDatabase(ds sqlutil.DataSource, peerID string) *DiscovererDatabase {
+func NewDON2DONDiscovererDatabase(ds sqlutil.DataSource, peerID string) *DiscovererDatabase {
```

Comment:
You could add a comment explaining that both DON2DON and OCR p2p comms are based on RageP2P library and bootstrap nodes but we want to keep their addresses separate to avoid any potential problems (e.g. one type of traffic starving the other one).